cellpack/autopack/utils.py

```diff
@@ -280,3 +303,1164 @@ def get_seed_list(packing_config_data, recipe_data):
         ]
 
     return seed_list
+
+
+# These functions have been moved over from Analysis.py
+"""
+Created on Mon May  6 22:58:44 2013
+
+@author: ludo
+"""
+
+
+def get_xyz_dict_from_all_pos_dict(all_pos_dict):
+    """
+    returns array of x, y, and z positions for each seed for runs
+    in all_pos_dict
+    """
+    all_objs = {}
+    for seed, object_dict in all_pos_dict.items():
+        for obj, positions in object_dict.items():
+            positions = np.array(positions)
+            if obj not in all_objs:
+                all_objs[obj] = {}
+            if seed not in all_objs[obj]:
+                all_objs[obj][seed] = {}
+            for ct, dim in enumerate(["x", "y", "z"]):
+                all_objs[obj][seed][dim] = positions[:, ct]
+    return all_objs
+
+
+def save_array_to_file(array_to_save, file_path, seed_index):
+    f_handle = open(file_path, "a" if seed_index else "w")
+    np.savetxt(
+        f_handle,
+        array_to_save,
+        delimiter=",",
+    )
+    f_handle.close()
+
+
+def getPositionsFromResFile(self):
+    # could actually restore file using histoVol.
+    # or not
+    # need to parse apr file here anyway
+    return []
+
+
+def getPositionsFromObject(self, parents):
```

Comment:
since this is no longer in a class, this call to self isn't going to reference anything. is this function still being used?